packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -373,4 +373,38 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
   closeTxs(): void {
     this.providers.ws.close('txs')
   }
+
+  async buildCustomTx(tx: BuildCustomTxInput): Promise<{
+    txToSign: ETHSignTx
+  }> {
+    try {
+      const { wallet, bip44Params = this.defaultBIP44Params } = tx
+
+      const from = await this.getAddress({ bip44Params, wallet })
+      const account = await this.getAccount(from)
+
+      const fees: Fees =
+        tx.maxFeePerGas && tx.maxPriorityFeePerGas
+          ? {
+              maxFeePerGas: numberToHex(tx.maxFeePerGas),
+              maxPriorityFeePerGas: numberToHex(tx.maxPriorityFeePerGas),
+            }
+          : { gasPrice: numberToHex(tx.gasPrice ?? '0') }
+
+      const txToSign: ETHSignTx = {
+        addressNList: toAddressNList(bip44Params),
+        value: tx.value,
+        to: tx.to,
+        chainId: 1,
```

Comment:
is this going to work for avax? `chainId` here should be `1` for eth mainnet and `43114` for avax c chain yes?

Reply:
It was working, but still shouldn't have been like this.

Addressed in https://github.com/shapeshift/lib/pull/1091/commits/bf4f47c99e91a65cf0994745adaa556311326df8